ML App UIKit/ML App UIKit/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  ML App UIKit
+//
+//  Created by Jose Chavez Aparicio on 20/10/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
Same comment about the extra spaces for the folder name, you can just call it UIKit

Reply:
Folder name updated